spaceflights/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/data_engineering/README.md

```diff
@@ -37,18 +34,18 @@ This modular pipeline preprocesses the raw data (`preprocessing_companies` and `
 |      |                    |
 | ---- | ------------------ |
 | Type | `pandas.DataFrame` |
-| Description | Cleansed and normalised version of the `companies` dataset |
+| Description | Preprocessed version of the `companies` dataset |
 
 ### `preprocessed_shuttles`
 
 |      |                    |
 | ---- | ------------------ |
 | Type | `pandas.DataFrame` |
-| Description | Cleansed and normalised version of the `shuttles` dataset |
+| Description | Preprocessed version of the `shuttles` dataset |
 
 ### `master_table`
 
 |      |                    |
 | ---- | ------------------ |
 | Type | `pandas.DataFrame` |
-| Description | A combined dataset from preprocessed companies and shuttle datasets, and source reviews data |
+| Description | A combined dataset containing data on shuttles, with company and reviews information joined in |
```

Comment:
Minor thing, just cause sth triggers me about sometimes about ending a sentence in a preposition. 😂  Feel free to ignore if not bothered.

```suggestion
| Description | A combined dataset containing data on shuttles, joined with company and reviews information |
```